presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraDistributed.java

```diff
@@ -129,4 +130,12 @@ public void testWrittenStats()
     {
         // Cassandra connector currently does not support create table nor insert
     }
+
+    @Override
+    public void testShowTablesLikeWithEscape()
+    {
+        //Cassandra connector currently does not support create table, so here override
+        MaterializedResult withEscape = computeActual("SHOW TABLES LIKE '%$_%' ESCAPE '$'");
+        assertThat(withEscape.getOnlyColumnAsSet()).allMatch(tableName -> ((String) tableName).contains("_"));
```

Comment:
Is there going to be _any_ such table? I mean: if result was empty (always and unconditionally due to some bug), the test would still pass.

If there isn't going to be any useful table for this test, leave this test method with just a comment, do not pretend something is being tested.